src/components/Header/HeaderDesktop.jsx

```diff
@@ -59,7 +59,7 @@ export const HeaderDesktop = () => {
                         alignItems: 'center',
                     }}
                     size="small"
-                    href="/"
+                    href="/#/vacancies"

```

Comment:
In development and production is not necessary use the **#** before the route

Reply:
Ok

Reply:
Yes is not neccessary.

Reply:
Ok I will leave it without the '#' but it will make some problems on deployment to GitHub pages. Maybe we need to change the deployment service